calculator.rb

```diff
@@ -0,0 +1,96 @@
+#create a calculator program that asks for user input and runs file from command line
+
+#define global scope variables
+results = nil
+
+#define all the operations to the user that is available to compute calculations
+operations_list = {
+"Addition": "+",
+"Subtraction": "-",
+"Multiplication": "*",
+"Division": "/"
+}
+
+#define a list with the various names for the operations to use as user may use abbreviated operation terms
+variations_list = ["addition", "add", "+", "subtraction", "subtract", "-", "multiplication", "multiply", "*", "division", "divide", "/"]
+
+#method for checking if value is numeric
+def is_number? string
+    true if Float(string) rescue false
```

Comment:
Nice! This method you defined is sooo useful and so elegant! In general, I may suggest an explicit return with `return true if Float(string) rescue false`, even though it ruins how elegant it looks. SORRYYY